src/Controller/Async/General.php

```diff
@@ -129,7 +130,12 @@ public function dashboardNews(Request $request)
         ];
 
         $response = $this->render('@bolt/components/panel-news.twig', ['context' => $context]);
-        $response->setSharedMaxAge(3600)->setPublic();
+        if ($response instanceof TemplateView) {
+            $this->setResponseCaching(3600);
+        } else {
+            /** @deprecated Deprecated since 3.3, to be removed in 4.0. */
+            $response->setSharedMaxAge(3600)->setPublic();
+        }
```

Comment:
Consolidated. 

I know this outcome from me, it is one where I think the problem most of the way though, then get partially side tracked in the process by a linked problem, come back and think I must have had it right so failing to take the thought pattern just that *one last step* :laughing: